packages/graphql/src/configs/localhost.js

```diff
@@ -14,14 +14,15 @@ const config = {
   ipfsGateway: `http://${HOST}:8080`,
   ipfsRPC: `http://${HOST}:5002`,
   relayer: `http://${HOST}:5100`,
-  bridge: 'https://bridge.dev.originprotocol.com',
+  bridge: `http://${HOST}:5000`,
```

Comment:
Can we revert this, so a local bridge is not required for local dev?

Reply:
Will revert it.